jupyterhub/templates/hub/service.yaml

```diff
@@ -19,9 +19,6 @@ metadata:
     {{- end }}
 spec:
   type: {{ .Values.hub.service.type }}
-  {{- with .Values.hub.service.loadBalancerIP }}
-  loadBalancerIP: {{ . }}
-  {{- end }}
```

Comment:
refactor: I moved this so it was positioned as it was in the other k8s Service templates